tic-tac-toe/src/components/info-bar/info-bar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './info-bar.css';
+
+const GameInfo = ({moves, onJumpTo, outerClassName}) => {
+    const innerClassName = "info-bar";
+    const className = `${outerClassName} ${innerClassName}`
+    const movesItem = moves.map((step, move) => {
+            return (
+                <li 
+                    className={innerClassName + "__item"}
```

Comment:
It's overkill here. Just add a className to your component.

Reply:
I fixed this.
```suggestion
<li key={move}>
```